training/src/main/java/wolox/training/models/Book.java

```diff
@@ -18,6 +22,9 @@
 /**
  * The class Book model.
  */
+@Getter
```

Comment:
Just a comment. In these cases, you can use @setter annotation too,  and add to the id field this property to ignore the setter implementation: @Setter(AccessLevel.NONE)

Reply:
Got it! But is there any trouble with the checkNoNull in the setters? or I just can delete them?

Reply:
BTW @Data is a good annotation too, because it contains  @ToString, @EqualsAndHashCode, @Getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor

Reply:
Ohh so Should I use @Data instead the annotations I used?

Reply:
I pushed the changes